openshift/che-monitoring.yaml

```diff
@@ -76,6 +76,7 @@ objects:
           - -upstream=${PROMETHEUS_SVC_URL}
           - -github-org=app-sre
           - -github-team=app-sre-observability
+          - --storage.tsdb.retention.time=30d
```

Comment:
This is the wrong container to add this to. 

We need to add it to the prometheus container, the one you've added it to is the reverse proxy that does authz/n sitting in front of prometheus

Reply:
good catch